### PR TITLE
Add privacy policy guide for TEC.

### PR DESCRIPTION
**Pull request by juanfra**
🎫 https://central.tri.be/issues/108454

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $descr is never used | [Privacy.php:39](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/Tribe/Privacy.php#L39)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:15](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L15)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:16](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L16)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:17](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L17)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:18](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L18)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:18](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L18)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:20](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L20)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:22](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L22)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:23](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L23)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$suggested_text' | [privacy.php:24](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L24)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [privacy.php:24](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L24)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:27](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L27)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:28](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L28)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:29](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L29)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:32](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L32)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:35](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L35)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:36](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L36)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:37](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L37)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:40](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L40)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:42](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L42)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:44](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L44)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:46](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L46)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$suggested_text' | [privacy.php:48](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L48)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [privacy.php:48](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L48)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:50](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L50)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:52](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L52)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:54](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L54)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:56](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L56)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:58](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L58)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:60](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L60)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [privacy.php:62](https://github.com/moderntribe/the-events-calendar/blob/93025c3910e188d62af994817f4671fc2c6b83e2/src/admin-views/privacy.php#L62)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $descr is never used | [Privacy.php:39](https://github.com/moderntribe/the-events-calendar/blob/f583abe3782f29bfa5e5a182cc3ee09e8043e424/src/Tribe/Privacy.php#L39)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$suggested_text' | [privacy.php:24](https://github.com/moderntribe/the-events-calendar/blob/f583abe3782f29bfa5e5a182cc3ee09e8043e424/src/admin-views/privacy.php#L24)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [privacy.php:24](https://github.com/moderntribe/the-events-calendar/blob/f583abe3782f29bfa5e5a182cc3ee09e8043e424/src/admin-views/privacy.php#L24)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$suggested_text' | [privacy.php:48](https://github.com/moderntribe/the-events-calendar/blob/f583abe3782f29bfa5e5a182cc3ee09e8043e424/src/admin-views/privacy.php#L48)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $descr is never used | [Privacy.php:39](https://github.com/moderntribe/the-events-calendar/blob/c0b985c5b2dcb42975c52fa01d6ea2a3a2a59303/src/Tribe/Privacy.php#L39)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$suggested_text' | [privacy.php:24](https://github.com/moderntribe/the-events-calendar/blob/c0b985c5b2dcb42975c52fa01d6ea2a3a2a59303/src/admin-views/privacy.php#L24)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$suggested_text' | [privacy.php:48](https://github.com/moderntribe/the-events-calendar/blob/c0b985c5b2dcb42975c52fa01d6ea2a3a2a59303/src/admin-views/privacy.php#L48)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $descr is never used | [Privacy.php:41](https://github.com/moderntribe/the-events-calendar/blob/e5c5aa0470a8313a46c7392d8a5f3d6631814702/src/Tribe/Privacy.php#L41)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$suggested_text' | [privacy.php:24](https://github.com/moderntribe/the-events-calendar/blob/e5c5aa0470a8313a46c7392d8a5f3d6631814702/src/admin-views/privacy.php#L24)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$suggested_text' | [privacy.php:48](https://github.com/moderntribe/the-events-calendar/blob/e5c5aa0470a8313a46c7392d8a5f3d6631814702/src/admin-views/privacy.php#L48)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Privacy.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	}
+
+	/**
+	 * Add the suggested privacy policy text to the policy postbox.
+	 *
+	 * @since TBD
+	 */
+	public function privacy_policy_content() {
+
+		if ( function_exists( 'wp_add_privacy_policy_content' ) ) {
```
[the remark this answers is not in the document]
Comment (juanfra) [reply src/Tribe/Privacy.php]: Sure

File: src/Tribe/Privacy.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	 * @param bool $descr Whether to include the descriptions under the section headings. Default false.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The default policy content.
+	 */
+	public function default_privacy_policy_content( $descr = false ) {
+
+		ob_start();
+		include_once Tribe__Events__Main::instance()->pluginPath . 'src/admin-views/privacy.php';
```
[the remark this answers is not in the document]
Comment (juanfra) [reply src/Tribe/Privacy.php]: Haven't used it before. We need to have in mind that this won't be only a tribe template, but instead, we'll be hooking onto a WP template with other plugins/WordPress core itself.

[the remark this answers is not in the document]
Comment (juanfra) [reply src/Tribe/Privacy.php]: @bordoni ☝️

**Pull request merged**